examples/language-sdk-instrumentation/golang-push/rideshare/prometheus-provisioning/prometheus.yaml

```diff
@@ -21,3 +21,6 @@ scrape_configs:
     static_configs:
       - targets:
           - host.docker.internal:9099
+  - job_name: 'rideshare'
+    static_configs:
+      - targets: ['us-east:5000', 'eu-north:5000', 'ap-south:5000']
```

Comment:
@marcsanmi this is why i need the container names , without this scrape i don't see the metrics i added show up in the explorer